charts/ci-operator/templates/steps/standard.yaml

```diff
@@ -86,6 +86,460 @@ step:
+---
+apiVersion: ci.w6d.io/v1alpha1
+kind: Step
+metadata:
+  name: static-test-nmapnsevulners
+  labels:
+  {{- include "ci-operator.labels" . | nindent 4 }}
+  annotations:
+    ci.w6d.io/kind: generic
+    ci.w6d.io/task: static-test
+    ci.w6d.io/order: "0"
+params:
+  - name: CLIENT_IP
+    description: defines the target ip or hostname
+    type: string
+  - name: CLIENT_PORT
+    description: defines the entry port
+    type: string
+  - name: CLIENT_PATHS
+    description: defines all path targeted
+    type: string
+    default: "/"
+step:
+  name: static-test-nmapnsevulners
+  image: "instrumentisto/nmap"
+  script: |
+    mkdir -p $(workspaces.source.path)/tests/nmap
+    cd $(workspaces.source.path)/tests
+    echo y | apk add --no-cache git
+    git clone https://github.com/vulnersCom/nmap-vulners.git vulnersCom_nmapvulners
+    nmap -Pn -v -A -T5 -p$(params.CLIENT_PORT) --script vulnersCom_nmapvulners/http-vulners-regex.nse --script-args paths={$(params.CLIENT_PATHS)} $(params.CLIENT_IP) -oX nmap/reportCVE-http-vulners-regex.xml
+    nmap -Pn -v -A -T5 --script vulnersCom_nmapvulners/vulners.nse --script-args mincvss=5.0 $(params.CLIENT_IP) -oX nmap/reportCVE-vulners.xml
+---
+apiVersion: ci.w6d.io/v1alpha1
+kind: Step
+metadata:
+  name: static-test-nmapnsevulscan
+  labels:
+  {{- include "ci-operator.labels" . | nindent 4 }}
+  annotations:
+    ci.w6d.io/kind: generic
+    ci.w6d.io/task: static-test
+    ci.w6d.io/order: "0"
+params:
+  - name: CLIENT_IP
+    description: defines the target ip or hostname
+    type: string
+  - name: CLIENT_PORT
+    description: defines the entry port
+    type: string
+step:
+  name: static-test-nmapnsevulscan
+  image: "instrumentisto/nmap"
+  script: |
+    mkdir -p $(workspaces.source.path)/tests/nmap
+    cd $(workspaces.source.path)/tests
+    echo y | apk add --no-cache git
+    git clone https://github.com/scipag/vulscan.git scipag_vulscan
+    nmap -Pn -v -A -T5 --script scipag_vulscan/vulscan.nse $(params.CLIENT_IP) -p$(params.CLIENT_PORT) -oX nmap/reportCVE-vulscan.xml
+---
+apiVersion: ci.w6d.io/v1alpha1
+kind: Step
+metadata:
+  name: build-sbt
+  labels:
+  {{- include "ci-operator.labels" . | nindent 4 }}
+  annotations:
+    ci.w6d.io/kind: generic
+    ci.w6d.io/task: build
+    ci.w6d.io/order: "0"
+params:
+  - name: COVERAGE
+    description: defines sbt coverage report config as true or false
+    type: string
+    default: "false"
+step:
+  name: build-sbt
+  image: "mozilla/sbt"
+  script: |
+    mkdir -p $(workspaces.source.path)/builds/sbt
+    cd $(workspaces.source.path)/builds
+    apt update
+    apt -y install git zip
+    git --version
+    git clone https://gitlab.com/jasperdenkers/lunatech-demo-ci-scala-sbt.git sbtapp
+    cd sbtapp
+    sbt clean test
+    if [[ "$(params.COVERAGE)" == "true" ]] ; then
+          sbt clean coverage test coverageReport
+    fi
+    cd target
+    zip -r ../../sbt/target.zip *
+    cd ../../sbt
+    ls target.zip
```

Comment:
hmmm
the project will be already set ready

Reply:
done